webapp/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityComposerDrawer.vue

```diff
@@ -281,6 +281,9 @@ export default {
         document.dispatchEvent(new CustomEvent('activity-composer-opened'));
       } else {
         document.dispatchEvent(new CustomEvent('activity-composer-closed'));
+        if (this.loading) {
+          this.loading = false;
+        } 
```

Comment:
Initialize `loading` in open method as any other `data` variable

```suggestion
        document.dispatchEvent(new CustomEvent('activity-composer-opened'));
      } else {
        document.dispatchEvent(new CustomEvent('activity-composer-closed'));
```

Reply:
done!